### PR TITLE
feat(arrow): adding arrow to go top

### PR DESCRIPTION
**Pull request by mfrachet**

### CODE REVIEW

**Comment by mfrachet** [comment]
https://github.com/Zenika/zenika.ca/issues/276

File: components/FloatingButton.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
```
Comment (lbonanni) [inline components/FloatingButton.vue]: it is a new component or can we use the already existing button component ?

Comment (mfrachet) [reply components/FloatingButton.vue]: The style is something really particular. I m not sure it's really good for the button to handle this kind of modifications. Anyway, I can move this in the button one

File: components/Arrow.vue [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="arrow">
+    <div class="bar left"></div>
```
Comment (lbonanni) [inline components/Arrow.vue]: maybe use the ^ character ?

File: layouts/default.vue [hunk 3]
```diff
@@ -29,5 +58,17 @@ export default {
   color: $primary;
   margin-top: 4em;
 }
+
+.button-go-top {
```
Comment (lbonanni) [inline layouts/default.vue]: floating button should handle this style

File: components/Arrow.vue (hunk 2 above)
Comment (mfrachet) [reply components/Arrow.vue]: screen shot 2019-02-06 at 11 11 03 am

**Pull request merged**